tokio/src/fs/mod.rs

```diff
@@ -108,6 +108,11 @@ where
     F: FnOnce() -> io::Result<T> + Send + 'static,
     T: Send + 'static,
 {
+    let handle = crate::runtime::Handle::current();
+    if handle.is_shutdown() {
+        panic!("Cannot run I/O operations on a runtime that has been shutdown");
+    }
+
```

Comment:
We want to keep all instances of these panic messages the same. See #3361.

Though in this case it probably should be panicking anyway inside the `sys::run` call. If it doesn't, fix `spawn_blocking` and this should fix itself.

Reply:
Do you mean that these panics should use `CONTEXT_MISSING_ERROR` as the message or that there should be constants for each message in `tokio/src/util/error.rs`?

Reply:
I don’t think this should panic. Hitting this is not due to a user bug. This probably should be an Err

Reply:
I'm not 100% sure 😆

Reply:
Ok, based on the Discord convo, FS ops will already error when the runtime shuts down (by returning `Err`). We should keep this behavior as is. We may want to look to see if the behavior is properly documented.